app/frontend/src/styles/application/jobseekers/search-panel.scss

```diff
@@ -16,13 +16,12 @@
   .govuk-link,
   .govuk-link:not(:focus),
   .govuk-label.govuk-label--s,
-  .js-location-finder__link {
+  .js-location-finder__link,
+  .signin--publisher {
     color: govuk-colour('white');
 
-    .govuk-link {
-      &:focus {
-        color: $govuk-focus-text-colour;
-      }
+    .govuk-link:focus {
```

Comment:
would rather this uses the `&:focus {` syntax - purely because we do everywhere else

Reply:
OK